source/frontend/src/features/documents/documentUpload/DocumentUploadContainer.tsx

```diff
@@ -125,7 +125,7 @@ export const DocumentUploadContainer = forwardRef<
       await Promise.all(uploadDocumentTasks);
     setIsUploading(false);
     if (tasksResult !== undefined) {
-      const batchResults = tasksResult.map<BatchUploadResponseModel>((r, index) => {
+      const batchResults = tasksResult.map((r, index) => {
```

Comment:
Why remove the type? typings are nice!

Reply:
I'll try to put it back, not quite the type I am expecting since this one is tightly coupled to Mayan responses.

Reply:
updated